docs/source/getting_started/input_output.md

```diff
@@ -143,17 +213,46 @@ save the data as separate single-animal DeepLabCut-style files.
 
 ::::{tab-item} LightningPose
 
-Save to LightningPose files (.csv).
+To save as a LightningPose file in .csv format:
 ```python
 save_poses.to_lp_file(ds, "/path/to/file.csv")
 ```
 :::{note}
-Because LightningPose saves pose estimation outputs in the same format as single-animal
-DeepLabCut projects, the above command is equivalent to:
+Because LightningPose follows the single-animal
+DeepLabCut .csv format, the above command is equivalent to:
 ```python
 save_poses.to_dlc_file(ds, "/path/to/file.csv", split_individuals=True)
 ```
 :::
 
 ::::
 :::::
+
+
+(target-saving-bboxes-tracks)=
+## Saving bounding boxes' tracks
```

Comment:
@niksirbi nice, I didn't realise `xarray` had `to_dataframe()` ✨ 

I had a go (with chatGPT providing 🐼 s help):

```python
df = ds.to_dataframe()

# Pivot
df_pivot = df.pivot_table(
    index=["time", "individuals"],
    columns="space",
    values=["position", "shape"],
).reset_index()


# Flatten the columns
df_pivot.columns = [
    "_".join(col).strip() 
    if (isinstance(col, tuple) and col[1]) 
    else col[0]
    for col in df_pivot.columns.values
]

# Rename columns to desired names
df_pivot = df_pivot.rename(
    columns={
        "position_x": "position_x",
        "position_y": "position_y",
        "shape_x": "width",
        "shape_y": "height",
    }
)

# Write to CSV
df_pivot.to_csv("test.csv", index=False)
```

I'm kinda inclined to leave the example using `csv`. My (maybe subjective) thinking is that:

(1) it seems that most common tracked bboxes output formats are csvs with different headings (see [MOT Challenge format](https://motchallenge.net/instructions/)), and the `csv` only approach seems a bit more straightforward to adapt to different headings.

(2) With the `csv` only approach  it kind of serves us as an additional example of how to index an `xarray` dataset.

(3) `csv` is part of the standard library (this is not a heavy reason, most of our users will know/use pandas).  

I am happy to add a couple of lines at the end giving some guides for this approach.

Let me know if you think otherwise, happy to discuss!

Reply:
I agree with you in this case. I had hoped that the `pandas` way would turn out to be less verbose, but your example shows that this is not the case. Happy for this to stay with `csv` for the reasons you mentioned. At some point we'll probably write a `save_bboxes` function anyway (to spare users from writing the `csv`/`pandas` code).